apps/web/src/components/FormList.tsx

```diff
@@ -18,21 +18,34 @@ import { FormInstanceEntity } from '@web/client';
 import { useState } from 'react';
 import { distance } from 'fastest-levenshtein';
 import { motion } from 'framer-motion';
+import Link from 'next/link';
+import { RightArrowIcon } from 'apps/web/src/static/icons';
+import { SearchAndSort } from 'apps/web/src/components/SearchAndRecent';
+import { ViewAll } from 'apps/web/src/components/ViewAll';
 
 /**
  * @param title - the title of the form list
  * @param formInstances - an array of form instances
  * @param color - the color of the form list
+ * @param extended - whether search and sort options are enabled
+ * @param link - link to page for category
+ * @param border - if true should have border
  * @returns a list of forms for the dashboard
  */
 export const FormList = ({
   title,
   formInstances,
   color,
+  extended,
```

Comment:
It looks like the `extended`, `link`, and `border` can be simplified such that `extended` and `border` is a single boolean like `isDashboard` and `link` can be an optional prop that's only required to be passed in if `isDashboard` is true (e.g. `{extended ? <SearchAndSort /> : <ViewAll title={title} link={link || '/'} />} and
```
export const FormList = ({
  title,
  formInstances,
  color,
  extended,
  link,
  border,
}: {
  title: string;
  formInstances: FormInstanceEntity[];
  color: string;
  isDashboard: boolean,
  link: string?
})
```